drivers/timer/stm32_lptim_timer.c

```diff
@@ -46,6 +46,10 @@ static const struct stm32_pclken lptim_clk[] = STM32_DT_INST_CLOCKS(0);
 
 static const struct device *const clk_ctrl = DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE);
 
+/* For tick accuracy, a specific tick to freq ratio is expected */
+/* This check assumes LSI@32KHz or LSE@32768Hz */
```

Comment:
Comment applies to a check while the next line is a definition. This is confusing